test/basic.test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import * as Basic from './cases/basic';
+import renderHelper from './renderHelper';
+
+describe('basic', () => {
+	it('renders correctly', () => {
+		const { element, rendered } = renderHelper(<Basic.HelloWorld/>);
+
+		expect(rendered).toMatchSnapshot();
+		expect(rendered).toEqual(element.childNodes[0]);
+	});
+
+	it('renders updates from render', () => {
```

Comment:
I would be great if it's possible to run the test suite twice, once with react and once with rawact. This way the snapshots and test cases would automatically validate against native react.

Reply:
great idea. done.